doc/whats_new.rst

```diff
@@ -59,6 +59,10 @@ Enhancements
     generated during build. Distribution packages will still contain generated
     C/C++ files. By `Arthur Mensch`_
 
+   - :class: `RobustScaler` now accepts ``quantile_range`` parameter.
+     (`#5929 <https://github.com/scikit-learn/scikit-learn/pull/5929>`_)
+     By `Konstantin Podshumok`_.
```

Comment:
you're creating a link here, so you should add a definition at the bottom of this file